### PR TITLE
Completing task5

### PR DESCRIPTION
**Pull request by ProkOksa**

### CODE REVIEW

File: Дз5/form.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      </label><br>
+      <label class="grade">
+        <input type="radio" name="gra">
+        <span>Плохо</span>
+      </label>
+    </div>
+    <div>
+      <p>Как вы проводите свое свободное время?</p>
+      <label class="leisure">
+        <input type="checkbox" name="lei">
```
Comment (dmitry-darly-solutions) [inline Дз5/form.html:41]: input and label should be on the same level

File: Дз5/form.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    <div>
+      <p>Логин</p>
+      <input type="text" placeholder="логин">
+      <p>Пароль</p>
+      <input type="password" placeholder="пароль">
+    </div>
+    <div>
+      <p>Оцените наш сайт</p>
+      <label class="grade">
+        <input type="radio" name="gra">
```
Comment (dmitry-darly-solutions) [inline Дз5/form.html:23]: image
image

File: Дз5/form.html [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+      <input type="text" placeholder="поиск" name="">
+    </div>
+    <div>
+      <p>Включить</p>
+      <div class="switch">
+        <div class="wrapper">
+          <div class="button" id="b1">
+            <input type="checkbox" class="checkbox">
+            <div class="position1"></div>
+            <div class="position2"></div>
```
Comment (dmitry-darly-solutions) [inline Дз5/form.html]: we discussed animation

File: Дз5/css/style_for_form.css [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+  width: 10px;
+  height: 10px;
+  border: 1px solid #adb5bd;
+  border-radius: 3px;
+  margin-right: 5px;
+  background-color: #FFFFFF;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: 70% 70%;
+}
```
Comment (valeriya-reva) [inline Дз5/css/style_for_form.css:83]: Better practice would be to combine all background properties to background.

File: Дз5/form.html [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+        <option value="Джаз">Джаз</option>
+        <option value="Кантри">Кантри</option>
+        <option value="Блюз">Блюз</option>
+      </select>
+    </div>
+    <div class="number">
+      <p>Количество</p>
+      <button class="number-minus" type="button" onclick="this.nextElementSibling.stepDown()">-</button>
+      <input type="number" min="0" value="0" readonly>
+      <button class="number-plus" type="button" onclick="this.previousElementSibling.stepUp()">+</button>
```
Comment (dmitry-darly-solutions) [inline Дз5/form.html:76]: image
not safe for browser support

File: Дз5/css/style_for_form.css [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+/*------Включить-----*/
+.switch {
+  position: relative;
+  width: 200px;
+  height: 50px;
+}
+
+.wrapper,
+.position1,
+.position2 {
```
Comment (valeriya-reva) [inline Дз5/css/style_for_form.css]: we can style it just with lable, without any div's
https://prnt.sc/tgn_UKPh3bdg
just use lable and depending on input, on checked or unchecked just change background and position before element

File: Дз5/form.html [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+          </div>
+        </div>
+      </div>
+    </div>
+    <div>
+      <p>Ваши интересы</p>
+      <textarea cols="30" rows="5"></textarea>
+    </div>
+    <div>
+      <input type="submit" value="Отправить">
```
Comment (dmitry-darly-solutions) [inline Дз5/form.html]: button

File: Дз5/css/style_for_form.css [hunk 8]
```diff
@@ -0,0 +1,187 @@
...
+  bottom: 0;
+  left: 0;
+}
+
+.button {
+  position: relative;
+  top: 50%;
+  width: 74px;
+  height: 36px;
+  margin: -20px auto 0 auto;
```
Comment (valeriya-reva) [inline Дз5/css/style_for_form.css]: not so good use margin-top
and there can be used more  structure
https://www.w3schools.com/howto/howto_css_switch.asp

File: Дз5/form.html [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+
+<body>
+  <h1>Форма</h1>
+  <hr>
+  <form action="/">
+    <div>
+      <p>Логин</p>
+      <input type="text" placeholder="логин">
+      <p>Пароль</p>
+      <input type="password" placeholder="пароль">
```
Comment (dmitry-darly-solutions) [inline Дз5/form.html]: all inputs in form should have name

File: Дз5/form.html (hunk 7 above)
Comment (valeriya-reva) [inline Дз5/form.html]: better to use tag button and should be with cursor pointer

**Review by dmitry-darly-solutions — APPROVED** [review]

**Pull request merged**